benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -14,6 +14,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseEventID": "processAudioVideoWelsh", "UserRole": "caseworker-sscs", "CRUD": "R"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseEventID": "uploadHearingRecording", "UserRole": "caseworker-sscs-clerk", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseEventID": "uploadHearingRecording", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
-  {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "dwpRequestHearingRecording", "UserRole": "caseworker-sscs-dwpresponsewriter", "CRUD": "CRUD"},
```

Comment:
Are we supposed to be removing permission for this dwpRequestHearingRecording event?

Reply:
No, that is an error. Will revert that